pkg/dao/interfaces.go

```diff
@@ -34,3 +34,12 @@ type ExternalResourceDao interface {
 	FetchSignature(url string) (*string, int, error)
 	FetchRepoMd(url string) (*string, int, error)
 }
+
+type MetricsDao interface {
+	RepositoriesCount() int
+	RepositoryConfigsCount() int
+	PublicRepositoriesNotIntrospectedLas24HoursCount() int
+	PublicRepositoriesFailedIntrospectionCount() int
+	NonPublicRepositoriesNonIntrospectedLast24HoursCount() int
+	Top50Repositories() int
+}
```

Comment:
I'm not sure if you planned to finish implementing these, but I suggest we leave additional metrics for another issue.

Reply:
:+1: I will add them into a new ticket; I forgot about the `interfaces.go`! thanks mate!

Reply:
I have implemented all but Top50Repositories, and I have added the remaining metric in a new ticket.